### ISSUE TITLE
TypeError: Cannot read property 'members' of null

### ISSUE THREAD

**Issue by Natemo6348** [galnir/Master-Bot#508]
If someone plays a song and loops it, then leaves the channel while it's still on loop, when song is over it logs this error:
```
/home/runner/Master-Bot/commands/music/play.js:518
  const rawMembers = Object.fromEntries(message.member.voice.channel.members);
                                                                     ^

TypeError: Cannot read property 'members' of null
    at interactiveEmbed (/home/runner/Master-Bot/commands/music/play.js:518:70)
```
bot then crashes completely

**Comment by galnir** [comment]
I'm able to reproduce this, looking into fixing it now

**Comment by galnir** [comment]
Fixed in e559683
Thanks for bringing this issue up 👍
